src/main/resources/db/migration/V4_0__create_indexes.sql

```diff
@@ -0,0 +1,32 @@
+-- ROLLBACK-START
+------------------
+-- DROP INDEX IX_MOTE_TILDELT_VEILEDER_IDENT;
+-- DROP INDEX IX_MOTE_TILDELT_ENHET;
+-- DROP INDEX IX_TID_STED_MOTE_ID;
+-- DROP INDEX IX_MOTEDELTAKER_ARBEIDSGIVER_MOTE_ID;
+-- DROP INDEX IX_MOTEDELTAKER_ARBEIDSGIVER_VIRKSOMHETSNUMMER;
+-- DROP INDEX IX_MOTEDELTAKER_ARBEIDSTAKER_MOTE_ID;
+-- DROP INDEX IX_MOTEDELTAKER_ARBEIDSTAKER_PERSONINDENT;
+-- DROP INDEX IX_MOTE_STATUS_ENDRET_MOTE_ID;
+-- DROP INDEX IX_MOTEDELTAKER_ARBEIDSTAKER_VARSEL_MD_AT_ID;
+---------------
+-- ROLLBACK-END
+
+CREATE INDEX IX_MOTE_TILDELT_VEILEDER_IDENT ON MOTE (tildelt_veileder_ident);
+
+CREATE INDEX IX_MOTE_TILDELT_ENHET ON MOTE (tildelt_enhet);
+
+CREATE INDEX IX_TID_STED_MOTE_ID ON TID_STED (mote_id);
+
+CREATE INDEX IX_MOTEDELTAKER_ARBEIDSGIVER_MOTE_ID ON MOTEDELTAKER_ARBEIDSGIVER (mote_id);
+
+CREATE INDEX IX_MOTEDELTAKER_ARBEIDSGIVER_VIRKSOMHETSNUMMER ON MOTEDELTAKER_ARBEIDSGIVER (virksomhetsnummer, mote_id);
+
+CREATE INDEX IX_MOTEDELTAKER_ARBEIDSTAKER_MOTE_ID ON MOTEDELTAKER_ARBEIDSTAKER (mote_id);
+
+CREATE INDEX IX_MOTEDELTAKER_ARBEIDSTAKER_PERSONINDENT ON MOTEDELTAKER_ARBEIDSTAKER (personident, mote_id);
```

Comment:
Motivasjonen for å ta med mote_id som en ekstra kolonne her er at jeg forventer at det kan bli laget et query der man joiner motedeltaker_arbeidstaker med mote-tabellen, men henter ut rader basert på personident. Litt usikker på om det er lurt å ta høyde for det.

Reply:
Sånn som ting ser ut nå er det ikke behov for henting av en liste med dialogmøter i kontekst av et en en virksomhet(`virksomhetsnummer`).

Reply:
OK, kan ta den bort. Men det skal ikke være så stort problem å ha noen ekstra indekser som foreløpig ikke dekker noe konkret behov.